pkg/query-service/pqlEngine/engine.go

```diff
@@ -23,7 +25,7 @@ type PqlEngine struct {
 
 func FromConfigPath(promConfigPath string) (*PqlEngine, error) {
 	// load storage path
-	c, err := pconfig.LoadFile(promConfigPath)
+	c, err := pconfig.LoadFile(promConfigPath, false, false, nil)
```

Comment:
why are we not using `ExternalLabels` unless they conflict in naming and won't be possible to query them?